Description: Fix use format argument
 Don't pass a non-literal string directly to fprintf as its format
 argument -- use "%s" instead.
Author: jonathanlennox <jonathanlennox>
Last-Update: 2011-05-26
--- a/crypto/math/math.c
+++ b/crypto/math/math.c
@@ -854,7 +854,7 @@
     *str-- = nibble_to_hex_char(tmp & 0xf);   
 
     /* now print stream */
-    fprintf(stream, string);
+    fprintf(stream, "%s", string);
   }
   
   return 0;
